### ISSUE TITLE
Potentially unused files [part 2]

### ISSUE THREAD

**Issue by c17johha** [HGustavs/LenaSYS#5976]
The following js/css/php/html files are not referenced by any other file (of those same types) on the site, check if they are still relevant (this list excludes /errorpages/*.php, which are referenced only by /.htaccess):
/DuggaSys/filereceive_preview.php
/DuggaSys/templates/3d-dugga.html
/DuggaSys/templates/3d-dugga.js
/DuggaSys/templates/bit-dugga.html
/DuggaSys/templates/bit-dugga.js
/DuggaSys/templates/boxmodell.html
/DuggaSys/templates/boxmodell.js
/DuggaSys/templates/clipping_masking_dugga.html
/DuggaSys/templates/clipping_masking_dugga.js
/DuggaSys/templates/color-dugga.html
/DuggaSys/templates/color-dugga.js
/DuggaSys/templates/curve-dugga.html
/DuggaSys/templates/curve-dugga.js
/DuggaSys/templates/default.js
/DuggaSys/templates/dugga1.html
/DuggaSys/templates/dugga1.js
/DuggaSys/templates/dugga2.html
/DuggaSys/templates/dugga2.js
/DuggaSys/templates/dugga3.html
/DuggaSys/templates/dugga3.js
/DuggaSys/templates/dugga4.html
/DuggaSys/templates/dugga4.js
/DuggaSys/templates/dugga5.html
/DuggaSys/templates/dugga5.js
/DuggaSys/templates/dugga6.html
/DuggaSys/templates/exampledugga.js
/DuggaSys/templates/exampledugga1.js
/DuggaSys/templates/exampledugga2.js
/DuggaSys/templates/exampledugga3.js
/DuggaSys/templates/exampledugga4.js
/DuggaSys/templates/exampledugga5.js
/DuggaSys/templates/exampledugga6.js
/DuggaSys/templates/exampledugga7.js
/DuggaSys/templates/feedback_dugga.html
/DuggaSys/templates/feedback_dugga.js
/DuggaSys/templates/generic_dugga_file_receive.html
/DuggaSys/templates/generic_dugga_file_receive.js
/DuggaSys/templates/html_css_dugga.html
/DuggaSys/templates/html_css_dugga.js
/DuggaSys/templates/html_css_dugga_light.html
/DuggaSys/templates/html_css_dugga_light.js
/DuggaSys/templates/kryss.html
/DuggaSys/templates/kryss.js
/DuggaSys/templates/placeholder_dugga.html
/DuggaSys/templates/placeholder_dugga.js
/DuggaSys/templates/shapes-dugga.html
/DuggaSys/templates/shapes-dugga.js
/DuggaSys/templates/transforms-dugga.html
/DuggaSys/templates/transforms-dugga.js
/DuggaSys/templates/XMLAPI_report1.html
/DuggaSys/templates/XMLAPI_report1_file_receive.js
/DuggaSys/testDugga.php
/Shared/BenchmarkLogging/index.html
/Shared/BenchmarkLogging/parseagents.php
/Shared/coursesyspw_place_two_levels_below.php
/Shared/css/blackTheme.css
/Shared/css/blank.css
/Shared/css/colorblind.css
/Shared/css/jquery.datetimepicker.css
/Shared/icons/animation.html

This is a child of: #5692

**Comment by a16joare** [comment]
I am not sure how to check this in a effective way.. Tried to search for the file name in atom to see if it loads somewhere and it gave no hits. I then searched for a file that i know is used but it did not gave any hits either, although i know where on the site the template is used. I removed the file from the project and then the possibility to create a dugga with that template was removed.  

Any suggestions how this can be done @HGustavs ?
